### PR TITLE
Switch to GitHub Actions

### LINKED ISSUE #89: Switch to GitHub actions

### PR DESCRIPTION
**Pull request by devpow112**
Should take care of #167, #166, #165 and #163 as well. Resolves #89.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,8 +23,8 @@
     "d2l-resize-aware": "github:BrightspaceUI/resize-aware#semver:^1",
     "d2l-users": "github:BrightspaceHypermediaComponents/users#semver:^2",
     "eslint-plugin-import": "^2.25.2",
-    "lit-element": "^3",
-    "lit-html": "^2"
+    "lit-element": "^2",
```
Comment (devpow112) [inline package.json:26]: These shouldn't be bumped yet. Added ignore for dependabot.

**Comment by devpow112** [comment]
I'll have to remove `Travis CI - Pull Request` as a required status check before merging this.

**Review by dlockhart — APPROVED** [review]
Thanks for catching the Lit bump.

File: .travis.yml [hunk 2]
```diff
@@ -1,28 +0,0 @@
...
-  target_commitish: master
-  on:
-    tags: true
-env:
-  global:
-  - OWNER_NAME=Brightspace
-  - REPO_NAME=awards-leaderboard-ui
-  - SAUCE_USERNAME: lweber-d2l
-  # GITHUB_RELEASE_TOKEN
-  - secure: U/gRDX/Ko8NuxoUALWtW3NfnLyf11CWDnRXUs0am+UQQNycUsAfhIRg45RxhHaNaUpuEsYR6k1Su/Xza+GVpJUIRVI58n6RvB+enKagRfad2Tfzhk24W3pq4CedU/BTMY3Ces0VI7H1s5hqcr2S29P2pDEAXtEM4Ndo0dovvOIZmZ27dkNz6zikqQj/s5/nM/MBi01+iHa04drqwmZdMXuPehohOAic4vN9eNSM/J5kXGqf8NRMdz7SQfdS+B+0O+2GSJj7IJYvinu0EGBc9Lljb0VQFR9WCqXPt4UWmDVUni068IKxW6h+ieZb/43QzpAFeYJ4xwIG7BxbjICnGiaglYMHNMEfbS40D1EMturVohXM9i4Wvm5c+1eUP6C11ZYlVkFq2fR3N5pAoZ+Jwfxlwb204ff9dKJ/e4BDBF5X1XpGTuSlqR+l1p16dlXIxGVDr5qpdFar52sGdL++skfo0qf5zeyspsySuMO/QlCQQCLelzCjC73Ie+01CWAVrzuoBlud49II1Jr9FACt5Pl//vTcgN0RwgyGwhazR951nEBag57K5LPjfU+BJYrzx8Zpmp16KXHE1FFZjR7OP8w1NKtPgdydR05HQh+bucw64W4VEekG4JChq6/ysXVB56D/zdBMivApBq2UHIJbLs7FKI6p4H6ZIhT7PW6uAfmg=
```
Comment (dlockhart) [inline .travis.yml:28]: I'll remove this access token from the bot account 🎉

Comment (devpow112) [reply .travis.yml:28]: Thanks

**Pull request merged**